build.gradle

```diff
@@ -27,33 +27,33 @@ apply plugin: 'signing'
 
 repositories mavenRepos
 
-dependencyManagement {
```

Comment:
I did this to test something back then. It can be put back now.

Reply:
Ok Reverted